### ISSUE TITLE
Which model of hard drive did you use?

### ISSUE THREAD

**Issue by phpdave11** [ewwhite/zfs-ha#4]
Can you recommend which hard drives to use?  I am trying to find the parts for "HP 4TB dual-port SAS hard drives".

I searched on HPE.com and could only find a dual-port SAS drive with 1.8TB (not 4TB).

https://www.hpe.com/us/en/product-catalog/servers/server-hard-drives.filters-facet_perf_url:Dual-Port%7Cfacet_interface_url:SAS%7Cfacet_optdrive_url:Hot-pluggable.html

By the way, thanks a lot for the write-up!  It was very helpful.

**Comment by ewwhite** [comment]
Well, you have two different drive form factors with HP. There are 2.5" (small form-factor) and 3.5" (large form-factor) disks. SFF drives are only available up to 1.8TB today. LFF disks reach 10TB today. So the drives I described in my build are HP 4TB 3.5" LFF SAS disks - part # 713850-B21

**Comment by phpdave11** [comment]
Thanks!

Those drives (#713850-B21) are no longer produced, right?  I want to buy new drives that are supported by a warranty.   

I think the filter on hpe.com does not work properly, because I was unable to find some of those drives after I selected "Dual-Port".

But after looking through the drives, I found this 10TB LFF SAS drive, part # 857644-B21, which appears to have dual-port SAS:
> Robust SAS interface (dual port, full duplex) for increased performance.

Do all **new** HP LFF SAS drives have dual-port SAS?
